spec/spec_helper.rb

```diff
@@ -0,0 +1,6 @@
+require "logstash/devutils/rspec/spec_helper"
+require 'logstash/devutils/rspec/logstash_helpers'
+
+require "logstash/filters/base"
+require "logstash/outputs/base"
+require "logstash/namespace"
```

Comment:
checking how much of them are ok to be removed and doing yay! love to clean up stuff